eng/common/instructions/azsdk-tools/local-sdk-workflow.instructions.md

```diff
@@ -117,3 +123,29 @@ Help the user generate and build SDKs locally from TypeSpec API specifications u
 **Actions**:
 
 - Run `azsdk_package_build_code` MCP tool to compile the SDK in the identified project directory.
+
+---
+
+### Step 3: Run package validation
```

Comment:
Same comment as above, this step needs to come after updating metadata and docs since the checks validate those changes.